bazel/grpc_deps.bzl

```diff
@@ -44,6 +49,11 @@ def grpc_deps():
         actual = "@upb//:json",
     )
 
+    native.bind(
+        name = "upb_generated_code_support__only_for_generated_code_do_not_use__i_give_permission_to_break_me",
+        actual = "@upb//:generated_code_support__only_for_generated_code_do_not_use__i_give_permission_to_break_me",
```

Comment:
This looks rather disturbing. But since we are checking in upb generated libraries, I guess we don't have other choices.

Reply:
I agree, this seems strange.

@haberman, is this okay to add?

Reply:
Note that we have decided to have this dependency anyway by virtue of checking in this code.

Reply:
Right, but as an implicit dependency, it could be changed without requiring changes on our end.  This way, it might, so I want to make sure that we're not opening the door to maintenance problems.

Reply:
As per our discussion, the only additional breakage this opens up is if the name of this target itself changes, which seems unlikely without making other changes that also affect us.  So I guess this is okay for the moment, but let's follow up on a better way to do this once Josh is back.